fv/named_ports_test.go

```diff
@@ -719,8 +719,8 @@ var _ = Describe("with a simulated kubernetes nginx and client", func() {
 			"tcp",
 		)
 		nginx.WorkloadEndpoint.Labels = map[string]string{
```

Comment:
and orchestrator

Reply:
oh hang on.  Is this using the main clientv2 or the backend client.

Assuming the former then we should remove the namespace label.
If the latter then we may need to add the orchestrator label as well.

Reply:
It's using the main v2 client, so we should just remove.